pycqed/measurement/pulse_sequences/fluxing_sequences.py

```diff
@@ -294,7 +294,7 @@ def chevron_seqs(qbc_name, qbt_name, qbr_name, hard_sweep_dict, soft_sweep_dict,
     flux_pulse['name'] = 'chevron_flux'
     flux_pulse['element_name'] = 'chevron_flux_el'
 
-    ro_pulses = generate_mux_ro_pulse_list([qbc_name, qbt_name],
+    ro_pulses = generate_mux_ro_pulse_list([qbr_name],
```

Comment:
I think it makes more sense to just have ro_pulses = deepcopy(operation_dict['RO ' + qbr_name]) rather than generate mux pulse list of one qubit. What do you think?

Reply:
Otherwise, looks good!

Reply:
The current version looks good to me

Reply:
Explanation: having a standard way to generate RO pulses, independent of the number of qubits is good.

Reply:
Ok then.